src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -10154,16 +10514,34 @@ namespace HVACUnitarySystem {
 		LoopOnOffFanPartLoadRatio = UnitarySystem( UnitarySysNum ).FanPartLoadRatio;
 		LoopCompCycRatio = UnitarySystem( UnitarySysNum ).CycRatio;
 
-		if ( CurOASysNum > 0 ) {
-			OASysEqSizing( CurOASysNum ).AirFlow = false;
-			OASysEqSizing( CurOASysNum ).Capacity = false;
-		} else if ( CurSysNum > 0 ) {
-			UnitarySysEqSizing( CurSysNum ).AirFlow = false;
-			UnitarySysEqSizing( CurSysNum ).Capacity = false;
-			AirLoopControlInfo( CurSysNum ).UnitarySysSimulating = false;
-		} else if ( CurZoneEqNum > 0 ) {
-			ZoneEqSizing( CurZoneEqNum ).AirFlow = false;
-			ZoneEqSizing( CurZoneEqNum ).Capacity = false;
+		if (  ! SysSizingCalc && UnitarySystem(UnitarySysNum).FirstPass ) {
+
```

Comment:
should have nested the if here and left only the bool as a check for the rest of the simulation.
if ( UnitarySystem(UnitarySysNum).FirstPass ) {
  if (  ! SysSizingCalc ) {
    UnitarySystem(UnitarySysNum).FirstPass = false;
  }
}

Just did this, will check in with next commit.